lib/rails_jmeter/ruby_jmeter/rails.rb

```diff
@@ -37,10 +41,15 @@ def assert_response(item, json_path: "", array_limit: nil)
           end
         else
           item = item.nil? ? "null" : item
-          assert json: json_path, value: item, name: "#{json_path} == #{item.to_json}"
+
```

Comment:
wait.... this is an `extract` (saving a variable from a regex), not an `assert` (comparing values), basically...... do you want to do:

https://github.com/flood-io/ruby-jmeter#response-assertion

```ruby
assert matches: regex.to_s
```

instead?